.eslintrc

```diff
@@ -139,21 +138,69 @@
                 " * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT",
                 " "
             ]
-        ]
+        ],
+        /*
+         * TODO: Address violations and re-enable these rules
```

Comment:
Amusingly we have a lint rule to ensure that `TODO`s always have an accompanying GitHub issue number... but looks like it doesn't check `.eslintrc`. 🙃

Reply:
Yeah, I've thought about adding a file extension to the file, so that it gets picked up by prettier/eslint, but I haven't bothered.